src/test/java/com/google/api/codegen/testdata/ruby/ruby_doc_library.baseline

```diff
@@ -2607,9 +2607,11 @@ module Library
           updater_proc = credentials.updater_proc
         end
 
+        package_version = Gem.loaded_specs['library'].version.version
```

Comment:
This looks wrong? Can you double check this?

Reply:
What about it?

Reply:
(If it's the double `version`, that's actually how it works... the first `.version` returns a `Gem::Version`, and the second returns a string.)